### ISSUE TITLE
DeLeveling from disconnection.

### ISSUE THREAD

**Issue by ffxi-meliadoul** [EdenServer/community#1408]
Hello, 

I was leveling THF in starter area outside Bastok and I fell asleep and was dc'd from the server by timeout. I awoke and logged back in and I had lost a level. 

No big deal, just a low level worth of xp.

So I started leveling again, got 2 levels up from what I lost. The level taken, and a new one.

Immediately after leveling again, I began a fight with a goblin and the client begins to not respond. I waited about 2 min and eventually got an error to close the program. Bummer.

When I came back online, I was again, knocked down a whole level from 8 to 7. Goblin is gone, I'm alive full health(thankfully lol) and I have 0 agro to anything around me.

A bit worried this is going to be a loop I'm in by logging out/timing out/leveling up/etc.

-Meliadoul

Forgot to mention, it deleted macro's I created as well, a simple Wasp Sting macro to be exact. Not a big deal, but was definitely proof to myself I wasn't crazy it was jumping back in time per say.

**Comment by Shoruto** [comment]
It might have to do with your internet connection having a little latency issue during that point.  Happens to the best of us.  Sometimes it can be the server also but usually it's internet based.  If you keep noticing a trend where you freeze up only when you are fighting, then it might be a server issue but for now just keep on trucking and you should be good :)

**Comment by ffxi-meliadoul** [comment]
Alrighty! Again, just worried I was about to be trapped in a loop tied to my character's unique ID or something along those lines. Trucking along is my goal :)

**Comment by ffxi-meliadoul** [comment]
It continually is happening in a certain part of the zone, no delevel this time, but its consistently making it hang up and kick me. Really glad it didn't wipe me this time, as I just went back and spent a lot of gil on AH. My new bow is still on me (whew!).

I know 2 things as far as my machine interacting with FFXI;
-My machine is way over powered for this, but cant rule out compatibility issues I suppose
-I've got roughly 40+ hours in and never had it do this, including this zone/area.

Going to close again, but I'm definitely avoiding this area for awhile. South Gustaberg (J/K/L-9)
